lib/ansible/galaxy/dependency_resolution/providers.py

```diff
@@ -5,6 +5,8 @@
 
 from __future__ import annotations
 
+from collections.abc import Sequence
```

Comment:
Since we often end up using many `collections.abc` imports for type hints, a common pattern we use is to import the entire module and alias it like we do the `typing` module:

```python
import collections.abc as c
```

You can use `_c` instead if you're taking care to avoid including imports in the public interface of a module.